code/controllers/subsystem/explosions.dm

```diff
@@ -121,25 +121,26 @@ SUBSYSTEM_DEF(explosions)
 		var/our_x = explode.x
 		var/our_y = explode.y
 		var/dist = CHEAP_HYPOTENUSE(our_x, our_y, x0, y0)
+		var/block = 0
 
 		if(newmode == "Yes")
 			if(explode != epicenter)
 				var/our_block = cached_exp_block[get_step_towards(explode, epicenter)]
-				dist += our_block
+				block += our_block
 				cached_exp_block[explode] = our_block + explode.explosive_resistance
 			else
 				cached_exp_block[explode] = explode.explosive_resistance
 
 		dist = round(dist, 0.01)
-		if(dist < dev)
+		if(dist + block < dev)
 			explode.color = "red"
 			explode.maptext = MAPTEXT("[dist]")
-		else if (dist < heavy)
+		else if (dist + block * 1.5 < heavy)
 			explode.color = "yellow"
-			explode.maptext = MAPTEXT("[dist]")
-		else if (dist < light)
+			explode.maptext = MAPTEXT("[dist + block * 1.5]")
+		else if (dist + block * 2.5 < light)
 			explode.color = "blue"
-			explode.maptext = MAPTEXT("[dist]")
+			explode.maptext = MAPTEXT("[dist + block * 2.5]")
```

Comment:
Could you use defines (def/undef within the file) for these multipliers?

Reply:
as in you want a variable to easily change them all from?

Reply:
i think i did it